### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by rebka1989** [DrCoffey/DeepSqueak#211] [General]
Installation tips:
"To run DeepSqueak, navigate to the main DeepSqueak folder in MATLAB, and type "DeepSqeak" into the command line. DeepSqueak will add itself to the MATLAB path after running."

Hello, I am trying to install DeepSqueak. I have Matlab open-- what is the "main DeepSqueak folder" and do I open it and then use the command line?

Thanks :)

**Comment by DrCoffey** [comment]
download DeepSqueak from github, open MATLAB, change the current folder in MATLAB to the main DeepSqueak folder (the folder that contains deepsqueak.m), then type DeepSqueak in the command line.

**Reply by rebka1989** [reply]
Thank you very much for the timely response :) I'm admittedly an ex-Luddite... 
I got it working! Best wishes.

**Comment by moosthuizen1** [comment]
I also have a problem getting DeepSqueak running. I followed the instructions, however I get a series of errors and I have no idea how to solve it. Any ideas? 

DeepSqueak version 3.1.0

Settings file not found. Create a new one...
Error using save
Unable to write file C:\Program Files\DeepSqueak\settings.mat: permission denied.

Error in squeakData/saveSettings (line 60)
            save(fullfile(obj.squeakfolder, 'settings.mat'), '-struct', 'settings')

Error in squeakData/loadSettings (line 69)
                saveSettings(obj)

Error in update_folders (line 5)
handles.data.loadSettings();

Error in DeepSqueak>DeepSqueak_OpeningFcn (line 178)
update_folders(hObject, eventdata, handles);

Error in gui_mainfcn (line 220)
    feval(gui_State.gui_OpeningFcn, gui_hFigure, [], guidata(gui_hFigure), varargin{:});

Error in DeepSqueak (line 29)
    gui_mainfcn(gui_State, varargin{:});
